src/annotator/components/Buckets.js

```diff
@@ -85,12 +86,14 @@ function NavigationBucketButton({ bucket, direction }) {
  *   @param {Bucket} props.below
  *   @param {Bucket[]} props.buckets
  *   @param {(annotations: AnnotationData[], toggle: boolean) => any} props.onSelectAnnotations
+ *   @param {(a: Anchor) => void} props.scrollToAnchor
```

Comment:
This typedef doesn't follow our typical preact-component conventions (usually we define `<ComponentName>Props` and don't use the prefixing/indentation variant.